.woodpecker/test.yml

```diff
@@ -6,9 +6,18 @@ when:
 
 steps:
   lint:
-    image: alpine/helm:latest
+    image: golang:alpine
     commands:
+      - apk add --no-cache git helm
       - helm lint --with-subcharts
+      ### helm-docs
+      # FIXME: revisit in when v1.11.1 is released
+      # until v1.11.1 is released
+      # - git clone https://github.com/norwoodj/helm-docs.git --depth 1
+      # - cd helm-docs/cmd/helm-docs && go build
+      # - mv helm-docs /usr/local/bin/
+      # - cd ../../.. && rm -rf helm-docs
+      # - helm-docs -x
```

Comment:
What would the additional lint do? 

If you want to check if the readmes got updated if needed by the PR author you could simply run `helm-docs` and fail on file changes using `git diff --exit-code`. However the docs would be currently updated by the CI on release automatically.

Reply:
`helm-docs` v1.11.1 has a new `-x` flag which asserts that new values are actually documented and that one has run `helm-docs` to update the README. 
This is currently not done and the reason why we are in an outdated state with the README compared to the actual values.

> However the docs would be currently updated by the CI on release automatically.

I see the workflow but I wonder if we should also do a PR lint to not fail during the release WF? If a param is missing/undocumented, this step will fail. This could be prevented during a PR lint upfront.

Reply:
If CI only updates during releases, the README bit be out-of-date quite often, i.e. every time there is no immediate release being issued after a PR is merged which does something to `values.yml`. This might be confusing for users?

We could update the README via CI on every push to `main` instead of only doing it after releases?

Reply:
I reverted the changes for now to simplify this PR but I think there is potential for improvement :)